groups/cdn/cloudfront.tf

```diff
@@ -53,6 +53,31 @@ resource "aws_cloudfront_origin_access_control" "assets" {
   signing_behavior                  = "always"
   signing_protocol                  = "sigv4"
 }
+resource "aws_s3_bucket" "log_bucket" {
+  bucket = "cloudfront-logs-bucket"
+
+  block_public_acls   = true
+  block_public_policy = true
+
+  lifecycle_rule {
+    id      = "log-retention"
+    enabled = true
+
+    expiration {
+      days = 30
+    }
+  }
+
+  tags = {
+    Name = "Cloudfront Logs Bucket"
+  }
+}
```

Comment:
The bucket name is too generic and should be named to better reflect the service it will be used for and to ensure global uniqueness.

Please also note that we have a naming scheme that should be followed to allow for namespaced buckets.

E.g.:  https://github.com/companieshouse/cdn-terraform/blob/main/groups/cdn/s3.tf#L1-L3

The `block_public_acls` and `block_public_policy` input args don't appear to be valid for the `aws_s3_bucket` resource.

I would strongly recommend reviewing the existing `s3.tf` file and look to add the logs bucket more in line with the configuration present there.